expansion/src/main/java/no/unit/nva/expansion/model/IndexDocumentWrapperLinkedData.java

```diff
@@ -56,14 +56,14 @@ public IndexDocumentWrapperLinkedData(UriRetriever uriRetriever) {
     public String toFramedJsonLd(JsonNode indexDocument) {
         var frame = SearchIndexFrame.FRAME_SRC;
         var inputStreams = getInputStreams(indexDocument);
-        return new FramedJsonGenerator(inputStreams, frame).getFramedJson();
+        return new FramedJsonGenerator(indexDocument, inputStreams, frame).getFramedJson();
     }
 
     //TODO: parallelize
 
     private List<InputStream> getInputStreams(JsonNode indexDocument) {
         final List<InputStream> inputStreams = new ArrayList<>();
-        inputStreams.add(stringToStream(toJsonString(indexDocument)));
+//        inputStreams.add(stringToStream(toJsonString(indexDocument)));
```

Comment:
Are you sure you want to remove this line? Is this not the persisted Publication? Not sure if test coverage in expansion is good enough to pick up if this is removing important data in the graph?

Reply:
Yes, we know read indexDocument and remove labels here:

```
public FramedJsonGenerator(JsonNode indexDocument, List<InputStream> streams, String frame) {
        removeAffiliationLabels(indexDocument);
        var indexDocumentInputStream = stringToStream(toJsonString(indexDocument));
        streams.add(indexDocumentInputStream);
        var model = createModel(streams);
        framedJson = frameJsonLd(model, frame);
    }
```